CovertInfiltration/Localization/BetterSecondWaveSupport.int

```diff
@@ -0,0 +1,3 @@
+[NoEnviromentalSitreps SecondWaveOptionObject]
+Tooltip="When enabled, missions will aquire SitReps only from plot/mission type and infiltration risks. Can be changed mid-campaign, but will not affect existing missions"
```

Comment:
"When enabled, missions will no longer receive environmental or third-party faction SitReps by random roll. This will effectively disable all environmental SitReps and third-party factions except for when the mission or map requires it. Since these SitReps are beneficial to neither XCOM nor ADVENT, the tactical difficulty of the game is unchanged. The positive and negative SitReps that appear randomly or from infiltration are not affected by this option. Can be changed mid-campaign, but will not affect existing missions."

Reply:
"When enabled, missions will no longer consistently receive environmental SitReps (such as extra explosive crates or extra psionic enemies) or third-party faction SitReps (such as the Lost or any mod-added factions). This will effectively disable all environmental SitReps and third-party factions except for when the mission or map requires it. Since these SitReps are generally beneficial to neither XCOM nor ADVENT, the tactical difficulty of the game is mostly unchanged. The spawning of positive or negative SitReps that tip mission difficulty are not affected by this option. Can be changed mid-campaign, but will not affect existing missions."

Reply:
> missions will no longer consistently receive environmental SitReps 

Hmmm, about that. I slapped a 50% chance on it, wasn't sure what to put there

Reply:
Let's change "consistently" to "sometimes" then